### ISSUE TITLE
Plains Pack 1 MonoMod incompatibility

### ISSUE THREAD

**Issue by nbusseneau** [jneb802/MoreWorldLocations_All#1]
Hey,

I am consistently reproducing this issue when connecting to a dedicated server with Plains Pack 1 installed on both the server and client sides:

```
06/12/2024 05:46:19: Console: [Info   : Unity Log] 06/12/2024 05:46:19: Exception in ZRpc::HandlePackage: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.IO.FileNotFoundException: Could not load file or assembly 'MonoMod.Backports, Version=1.1.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies.
  at (wrapper managed-to-native) System.Object.__icall_wrapper_mono_gc_wbarrier_generic_nostore_internal(intptr)
  at (wrapper write-barrier) System.Object.wbarrier_conc(intptr)
  at (wrapper managed-to-native) System.Reflection.RuntimeMethodInfo.InternalInvoke(System.Reflection.RuntimeMethodInfo,object,object[],System.Exception&)
  at System.Reflection.RuntimeMethodInfo.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x0006a] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
   --- End of inner exception stack trace ---
  at System.Reflection.RuntimeMethodInfo.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00083] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
  at System.Delegate.DynamicInvokeImpl (System.Object[] args) [0x000e7] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
  at System.MulticastDelegate.DynamicInvokeImpl (System.Object[] args) [0x00008] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
  at System.Delegate.DynamicInvoke (System.Object[] args) [0x00000] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
  at ZRpc+RpcMethod`1[T].Invoke (ZRpc rpc, ZPackage pkg) [0x0001d] in <a56bd2e63844419798a70e81e689ed8a>:0
  at (wrapper dynamic-method) ZRpc.DMD<ZRpc::HandlePackage>(ZRpc,ZPackage)
  at ZRpc.Update (System.Single dt) [0x0003e] in <a56bd2e63844419798a70e81e689ed8a>:0

06/12/2024 05:46:19: Exception in ZRpc::HandlePackage: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.IO.FileNotFoundException: Could not load file or assembly 'MonoMod.Backports, Version=1.1.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies.
  at (wrapper managed-to-native) System.Object.__icall_wrapper_mono_gc_wbarrier_generic_nostore_internal(intptr)
  at (wrapper write-barrier) System.Object.wbarrier_conc(intptr)
  at (wrapper managed-to-native) System.Reflection.RuntimeMethodInfo.InternalInvoke(System.Reflection.RuntimeMethodInfo,object,object[],System.Exception&)
  at System.Reflection.RuntimeMethodInfo.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x0006a] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
   --- End of inner exception stack trace ---
  at System.Reflection.RuntimeMethodInfo.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00083] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
  at System.Delegate.DynamicInvokeImpl (System.Object[] args) [0x000e7] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
  at System.MulticastDelegate.DynamicInvokeImpl (System.Object[] args) [0x00008] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
  at System.Delegate.DynamicInvoke (System.Object[] args) [0x00000] in <17d9ce77f27a4bd2afb5ba32c9bea976>:0
  at ZRpc+RpcMethod`1[T].Invoke (ZRpc rpc, ZPackage pkg) [0x0001d] in <a56bd2e63844419798a70e81e689ed8a>:0
  at (wrapper dynamic-method) ZRpc.DMD<ZRpc::HandlePackage>(ZRpc,ZPackage)
  at ZRpc.Update (System.Single dt) [0x0003e] in <a56bd2e63844419798a70e81e689ed8a>:0
```

This does not happen when disabling Plains Pack 1 on the server side.
I think the build of Plains Pack 1 that just got uploaded is using an old version of the Jotunn package:

https://github.com/jneb802/MoreWorldLocations_All/blob/7a1f484996eb5b9d9c822b2f6acf38231465498b/Plains%20Pack%201/Plains%20Pack%201.csproj#L156

than for example Mistlands Pack 1 which uses the latest one:

https://github.com/jneb802/MoreWorldLocations_All/blob/7a1f484996eb5b9d9c822b2f6acf38231465498b/Mistlands%20Pack%201/Mistlands%20Pack%201.csproj#L155

(Yet funnily enough it has more recent MonoMod packages, which might also be causing the issue:

https://github.com/jneb802/MoreWorldLocations_All/blob/7a1f484996eb5b9d9c822b2f6acf38231465498b/Plains%20Pack%201/Plains%20Pack%201.csproj#L158-L160

vs. for example:)

https://github.com/jneb802/MoreWorldLocations_All/blob/7a1f484996eb5b9d9c822b2f6acf38231465498b/Mistlands%20Pack%201/Mistlands%20Pack%201.csproj#L157-L159

**Comment by nbusseneau** [comment]
This was fixed in release 1.0.1, cheers :)

**Comment by jneb802** [comment]
Thanks for reporting. You were correct that the issue was caused by an older version of Jotunn.
